doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -27,6 +29,13 @@ Here are a few examples:
  * "HTML" in PascalCase is HTML
  * "HTML" in camelCase is html
 
+Exceptions:
+ The following are exceptions to the above capitalization guidelines that occur
+ in the standard modules or packages:
+
+  * :proc:`~ChapelLocale.locale.numPUs` (num Processor-Units)
+
+
```

Comment:
I didn't see this PR before it got merged, but I might be inclined to put a statement here along the lines of.  "Our hope and intention is to update our rules over time so that they cover cases like this, obviating the need to call it out as an exception."  (i.e., indicate that we're not content to have exceptions).  If we thought we needed it we could say "At this time, we don't yet know which of several candidate rules might be used to explain the exception." — but maybe that's expressing a bit more doubt than we're comfortable with?).

Reply:
I did not think of that, but I like the idea.

What about: "Our hope and intention is to update our rules over time so that they cover cases like this, obviating the need to call it out as an exception. As more exceptions accumulate, rules may be amended to cover multiple exceptions at once" ?

Reply:
That sounds fine to me.  We could also have @riftEmber give it a look as a third pair of eyes and your original reviewer.

Reply:
Yes, I would be in favor of that wording.

Reply:
Great, I'll create an update for this PR with that change.